numpy/__init__.pxd

```diff
@@ -182,9 +182,8 @@ cdef extern from "numpy/arrayobject.h":
         NPY_ARRAY_UPDATE_ALL
 
     cdef enum:
-        NPY_MAXDIMS
-
-    npy_intp NPY_MAX_ELSIZE
```

Comment:
Maybe a bit random, but `NPY_MAX_ELSIZE` doesn't exist (it sure did at some point).

Reply:
Does this removal need a release note? Just to say that if someone was using and relying on it, it wasn't doing what they thought it did.